src/agent/src/config.rs

```diff
@@ -110,6 +111,7 @@ pub struct AgentConfig {
     pub guest_components_procs: GuestComponentsProcs,
     #[cfg(feature = "guest-pull")]
     pub image_registry_auth: String,
+    pub data_integrity: bool,
```

Comment:
`data_integrity` is a pretty generic term. Maybe this should be something more specific to secure storage?

Reply:
I have renamed `data_integrity` to `secure_storage_integrity`.